src/SDK/SmartDetectorsPackage/SmartDetectorsPackage.nuproj

```diff
@@ -22,7 +22,7 @@
   <Import Project="$(NuProjPath)\NuProj.props" Condition="Exists('$(NuProjPath)\NuProj.props')" />
   <PropertyGroup Label="Configuration">
     <Id>Microsoft.Azure.Monitoring.SmartDetectors</Id>
-    <Version>0.11.1.0</Version>
+    <Version>0.12.0.0</Version>
```

Comment:
Please also update smart detector version in project templates.

Reply:
this is a separate task

---
In reply to: [235015949](https://github.com/Azure/SmartDetectorsMonitoringAppliance/pull/27#discussion_r235015949) [](ancestors = 235015949)